myriad-scheduler/src/test/java/org/apache/myriad/configuration/MyriadConfigurationTest.java

```diff
@@ -21,71 +21,51 @@
+public class MyriadConfigurationTest extends BaseConfigurableTest {
 
-  static MyriadConfiguration cfg;
+  public void testMyriadContainerConfiguration() throws Exception {
+    MyriadContainerConfiguration conf = cfgWithDocker.getContainerInfo().get();
+    assertTrue(conf.getDockerInfo().isPresent());
 
-  @BeforeClass
-  public static void setUpBeforeClass() throws Exception {
-    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
-    cfg = mapper.readValue(Thread.currentThread().getContextClassLoader().getResource("myriad-config-test-default.yml"),
-        MyriadConfiguration.class);
+    MyriadDockerConfiguration dConf = conf.getDockerInfo().get();
+    assertEquals(false, dConf.getForcePullImage());
+    assertEquals("mesos/myriad", dConf.getImage());
 
-  }
-
-  @Test
-  public void serviceConfigurationTest() throws Exception {  
-    Map<String, ServiceConfiguration> auxConfigs = cfg.getServiceConfigurations();
-
-    assertEquals(auxConfigs.size(), 2);
-
-    for (Map.Entry<String, ServiceConfiguration> entry : auxConfigs.entrySet()) {
-      String taskName = entry.getKey();
-      ServiceConfiguration config = entry.getValue();
-      String outTaskname = config.getTaskName();
-      assertEquals(taskName, outTaskname);
+    assertNotNull(conf.getVolumes());
+    
+    Set<String> keys = Sets.newHashSet("hostPath", "containerPath", "mode");
+    Set<String> modes = Sets.newHashSet("RO", "RW");
+    Iterator<Map<String, String>> iter = conf.getVolumes().iterator();
+    
+    while (iter.hasNext()) {
+      Map<String, String> mcConf = iter.next();
+      assertEquals(keys, mcConf.keySet());
+      assertTrue(modes.contains(mcConf.get("mode")));
     }
   }
-
+  
   @Test
-  public void coreConfigurationTest() throws Exception {
-    assertEquals("MyriadTest", cfg.getFrameworkName());
-
-    //authorization parameters
+  public void testRoles() throws Exception {
+    assertEquals("test", cfgWithRole.getFrameworkRole());
     assertEquals("*", cfg.getFrameworkRole());
-    assertEquals("hduser", cfg.getFrameworkUser().get());
-    assertEquals("root", cfg.getFrameworkSuperUser().get());
-
-    //ports and directory paths
-    assertEquals("10.0.2.15:5050", cfg.getMesosMaster());
-    assertEquals("/usr/local/lib/libmesos.so", cfg.getNativeLibrary());
-    assertEquals(new Integer(8192), cfg.getRestApiPort());
-    assertEquals("10.0.2.15:2181", cfg.getZkServers());
-  
-    //timeouts
-    assertEquals(new Double(44200000), cfg.getFrameworkFailoverTimeout());
-    assertEquals(new Integer(25000), cfg.getZkTimeout());
-  
-    //checkpoints
-    assertEquals(false, cfg.isCheckpoint());
-    assertEquals(true, cfg.isHAEnabled());
-    assertEquals(false, cfg.isRebalancerEnabled());
   }
-  
+
   @Test
-  public void executorConfigurationTest() throws Exception {
+  public void testExecutorConfiguration() throws Exception {
     MyriadExecutorConfiguration conf = cfg.getMyriadExecutorConfiguration();
```

Comment:
I do not think this change is necessary, as usage of annotations allows to have any naming for the tests.